src/sdk/glacier_tapecloud.js

```diff
@@ -425,6 +453,25 @@ class TapeCloudGlacier extends Glacier {
         return result.toLowerCase().trim() === 'true';
     }
 
+    /**
+     * 
+     * @param {string} data 
+     * @returns 
```

Comment:
Can you explain how the line with look like after encoding maybe also give an example.

Reply:
Added an example and some description to the method.